src/main/java/backend/UpdateController.java

```diff
@@ -61,6 +62,15 @@ public void processAndRefresh(List<FilterPanel> filterPanels) {
                     if (toUpdate.isEmpty()) {
                         // If no issues requiring metadata update, just run the filter and sort.
                         logic.updateUI(processFilters(filterExprs));
+                        ArrayList<CompletableFuture<Boolean>> stub = new ArrayList<>();
+                        Futures.sequence(stub)
+                                .thenRun(() -> {
+                                    if (isResumeTimer) {
+                                        logic.refreshTimer.resume();
+                                    }
+                                })
+                                .thenCompose(n -> logic.getRateLimitResetTime())
+                                .thenApply(logic::updateRateLimits);
```

Comment:
I'm a bit uneasy about this extra complications added to this method. May be @ndt93 can weigh in? or @acruis who wrote the API quote monitor earlier?